### DISCUSSION TITLE
Database changes Migration from local to production environment

### DISCUSSION THREAD

**Discussion by msreekm** [strapi/strapi#6025] [Legacy Topics]
Wondering how people are migrating datbase changes from dev to prod , api changes are checked into git , how do we track and migrate fields that got added,updated,removed in db? Wondering how the community is doing this now ? Iam using postgres as db.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by msreekm** [reply]
thanks

**Comment by lauriejim** [comment]
It's not an easy thing to do, you will have to import/export manually some database information?
The database structure is automatically sync, based on your files too.
For permissions, etc... you will have to do it manually.
Here is an item we will work on - https://portal.productboard.com/strapi/1-public-roadmap/c/12-migrate-across-environments
